aws/table_aws_ec2_managed_prefix_list.go

```diff
@@ -22,7 +22,10 @@ func tableAwsEc2ManagedPrefixList(_ context.Context) *plugin.Table {
 		Description: "AWS EC2 Managed Prefix List",
 		List: &plugin.ListConfig{
 			IgnoreConfig: &plugin.IgnoreConfig{
-				ShouldIgnoreErrorFunc: shouldIgnoreErrors([]string{"InvalidAction", "InvalidRequest"}),
+				// We are dealing with the UnsupportedOperation error code as a result of the resource's unsupported area.
+				// The specified request includes an unsupported operation. For example, you can't stop an instance that's instance store-backed. Or you might be trying to launch an instance type that is not supported by the specified AMI. The returned message provides details of the unsupported operation.
+				// We need to add the 'UnsupportedOperation' error code to the IgnoreErrorCodes function because while the EC2 service is supported in the 'me-south-1' region, listing managed prefix lists is not supported in that region.
```

Comment:
```suggestion
				// Ignore the 'UnsupportedOperation' error because while the EC2 service is supported in the 'me-south-1' region, listing managed prefix lists is not in that region
```

Updated suggestion that's a little less wordy and removes unrelated info about what `UnsupportedOperation` could mean in other tables.